.drone.yml

```diff
@@ -179,3 +179,18 @@ pipeline:
       branch: master
       event: push
       status: failure
+
+  promote_to_uat:
+    image: quay.io/ukhomeofficedigital/drone-trigger:latest
+    secrets:
+      - drone_token
+    drone_server: https://drone-gitlab.acp.homeoffice.gov.uk
+    drone_token: $${DRONE_TOKEN}
+    repo: lev/levops
+    branch: master
+    event: deployment
```

Comment:
The event will be a push. (Though that might be the default so perhaps you can remove the line.)